modules/oe_bootstrap_theme_paragraphs/config/install/field.storage.paragraph.field_oe_links_block_background.yml

```
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_oe_links_block_background
+field_name: field_oe_links_block_background
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: gray
+      label: gray
```

```suggestion
      label: Gray
```
Labels should have the first letter capitalised.